packages/components/organisms/f-checkout/src/components/Selector.vue

```diff
@@ -77,6 +78,10 @@ export default {
          */
         shouldShowPreOrderWarning () {
             return !this.availableFulfilment.isAsapAvailable;
+        },
+
+        shouldShowSelector () {
+            return this.serviceType !== 'dinein' || this.availableFulfilment.times.length > 1;
```

Comment:
Is it guaranteed that there will be no `availableFulfilment` times if the service type is dine-in?

Reply:
Sounds good, as long as it has been discussed I'm happy.